examples/notebooks/mnist.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
Updated with real numbers  -- you can see the .999 auc for all the digits.
https://github.com/lilleswing/deepchem/blob/notebook-tests-more-cr/examples/notebooks/mnist.ipynb